lib/mwUtil.js

```diff
@@ -33,6 +33,15 @@ mwUtil.makeETag = (rev, tid, suffix) => {
  * Normalizes the request.params.title and returns it back
  */
 mwUtil.normalizeTitle = (hyper, req, title) =>  mwUtil.getSiteInfo(hyper, req)
+.catch((e) => {
```

Comment:
The SiteInfo is used in various other places, so a better spot to do that is in the action module, here: https://github.com/wikimedia/restbase/blob/master/sys/action.js#L266